sympy/polys/polymatrix.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import print_function
+
+from sympy.core import S
+from sympy.matrices.dense import DenseMatrix
+from sympy.matrices.matrices import classof
+
+
+class PolyMatrix(DenseMatrix):
+    """
+    Matrix of polynomials.
```

Comment:
There could also be other polys' objects, e.g., fractions. I'd think of 'Poly` as referring to everything in `polys`.